modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/getTooltipTitles.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+export function getTooltipTitles({title}) {
```

Comment:
If we are using just the title, should be good pass only the title instead destruct a object.

```suggestion
export function getTooltipTitles(title) {
```

Reply:
the contentRenderer from ClayTooltipProvider only returns title from props, see here: https://clayui.com/docs/components/tooltip/api.html

Reply:
Sure. But I guess that should be better destructure the object before and pass only the title as param for this function.